index.js

```diff
@@ -214,6 +214,8 @@ function $asNull () {
 function $asInteger (i) {
   if (isLong && isLong(i)) {
     return i.toString()
+  } else if (typeof i === 'bigint') {
```

Comment:
This library currently tests all the way back to Node 6 -- https://github.com/fastify/fast-json-stringify/blob/4b6df2e11c4f5a8b2c408741506f48c3f589865a/azure-pipelines.yml#L5-L13

What will this line do on < 10.13?

Reply:
Nothing, it's just a string comparison, which means that it will always be false.
Anyhow, we will drop Node v6 and v8 once we release Fastufy v3 :)

Reply:
I know. But it still has to work for the versions this will actually target.

Reply:
It does, as you can see all the tests are passing :)
You can use the BigInt syntax only from Node 10.4 and above.
In any other case the typeof will return something else, so the if will never be true.